CMakeLists.txt

```diff
@@ -378,6 +378,9 @@ install(
     DESTINATION
         "${PDAL_LIB_INSTALL_DIR}/cmake/PDAL")
 include(${PDAL_CMAKE_DIR}/config.cmake)
+target_include_directories(${PDAL_BASE_LIB_NAME}
+    INTERFACE
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include>)
```

Comment:
Do we need the same for pdal_util as well?

Reply:
Not need.  The only reason is if you're using the utilities without the main PDAL lib.  But I can add it.

Reply:
we should do so in case someone follows a pattern from a PDAL filter/reader in their own plugin or something.